tests/test_main.py

```diff
@@ -776,6 +776,7 @@ def test_sample_files(self, full_filename, result_filename):
 
 
     def test_module_files(self):
+        pytest.skip("this test is not deterministic and can cause random ci failiurs. We can enable it later again")
```

Comment:
I'm assuming this is for the problem discussed in https://github.com/alexmojaki/executing/pull/74#issuecomment-2238231857. Please try narrowing this skip to the affected files and Python versions.